src/include/security/routines.rb

```diff
@@ -89,7 +89,7 @@ def settings2widget(_ID)
       # "Widget" == "IntField"
       if widget == "IntField"
         intval = Builtins.tointeger(value)
-        intval = 0 if intval == nil
+        intval = 0 if intval.nil?
```

Comment:
I know, as my proposal won't work if intval will be boolean, but it is not, so it is safer and nicer.